docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -1,53 +1,167 @@
+| **Semgrep support portal** | ❌ no | ✔️  yes | ✔️  yes |
+| **Customer success manager** | ❌ no | ❌ no | ✔️  yes |
+| **Dedicated and customized onboarding** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* Request support and track the status of your tickets through the [Semgrep Support Portal](https://support.semgrep.dev/hc/en-us), which is available 8 hours a day, 5 days a week.
+
+
+## Determining your plan needs
+
+### Number of seats
+
+Within your team or organization, assess the number of members whose code needs scanning. That will determine the number of seats needed for purchase.
+
+For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on their merge or pull requests, only 4 seats are required even if the organization has a total of 10 members. If these unique contributors commit to many projects within the same organization, they will only be counted once, so no additional seats are required.
+
+All members of the organization, regardless of seat status, have access to paid features for the chosen tier. This means that project managers and other non-programming roles can still view the Semgrep App dashboard.
+
+### Plan limits
+
+Should your plan exceed the limit, we will reach out and come up with a new plan that fits your needs.
 
-Yes, you can use the Semgrep CLI and Semgrep Community Tier for [free](https://semgrep.dev/getting-started)!
+## Upgrading your plan
 
-## What are the differences between the free and paid plans?
+To upgrade to the **Team tier** through **credit card**:
 
-Semgrep Community is great for individuals looking to quickly secure their public and private projects and teams looking to explore and try out the product.
+1. In the Settings page, select the Payment tab.
+2. Select the number of seats to purchase.
+3. Fill in your payment details.
 
-Semgrep Team and Enterprise are best for larger organizations that require dedicated support and advanced functionality.
 
-You can find more on the [pricing page](https://semgrep.dev/pricing).
+![Screenshot of Semgrep App payment menu](../img/billing-and-pricing-payment.png)<br />
 
-## How many seats do I need?
+To upgrade to the **Enterprise tier**, contact us through our [Semgrep for organizations](https://semgrep.dev/contact) page.
 
-One seat corresponds to one project contributor whose merge/pull request or code is scanned during the billing period.
+### Billing
 
-For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on their merge/pull requests, 4 seats are required. If these unique contributors commit to many projects within the same organization, they will only be counted once, so no additional seats are required.
+Team tier users are charged monthly. Enterprise tier users are charged at an agreed-upon billing cycle. For any concerns, send an email to [billing@r2c.dev](mailto:billing@r2c.dev) to get in touch with our sales team.
```

Comment:
Re monthly charges, that’s true for the team tier users who pay via credit card (but not all team tier users pay that way).

Reply:
Have corrected this. Thank you.